### ISSUE TITLE
Google Play window showing up empty

### ISSUE THREAD

**Issue by Frozenfire92** [tiliado/nuvolaplayer#207]
I've had this happen numerous times, it seems to fix itself by reopening a dozen or so times

![image](https://cloud.githubusercontent.com/assets/5893112/13007205/fe71b314-d164-11e5-9415-bdc20d8f6954.png)

**Comment by fenryxo** [comment]
Hello @Frozenfire92. Thanks for taking the time to report your issue. I need more information to investigate it further:

* Version of Nuvola Player and your syste,
* Debugging output from terminal when the issue happens.

See [Bug Reporting Guidelines](https://github.com/tiliado/nuvolaplayer/wiki/Bug-Reporting-Guidelines) for details.

**Comment by Frozenfire92** [comment]
Sorry for the lack of information!

### System
```
Distributor ID:	Ubuntu
Description:	Ubuntu 15.10
Release:	15.10
Codename:	wily
x86_64
Desktop Environment: Gnome (Official Ubuntu Gnome flavour)
```

### Nuvola
**Note** It updated today, but the version it initally occursed was what was before (assuming 3.0.0). I was able to reproduce it however
``` 
Nuvola Player 3 3.0.1
Revision 018a1322200541ee7c378b2418d497ddd17bae15
Installed via Tiliado Repository Installer
```

### Reproduce 
1. Open Google play directly or through the launcher
2. Observer screenshot above

I was able to produce it with `nuvolaplayer3 --debug` and got the following output
```bash
Master: [DEBUG      Gtk] Connecting to session manager
Master: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: gnome shell 3.16.4, CSD 1, appmenu 1, menubar 0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.3
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.12
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_8tracks 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_amazon_cloud_player 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_bandcamp 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_deezer 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_jango 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_kexp 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_logitech_media_server 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_mixcloud 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_plex 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_spotify 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_test 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_tunein 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_yandex_music 48 not found.
Master: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.3
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.12
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [INFO     Nuvola] WebAppRegistry.vala:89: Using web app Google Play Music, version 5.12, data dir /usr/share/nuvolaplayer3/web_apps/google_play_music
Master: [DEBUG    Nuvola] MasterController.vala:463: Launch app runner for 'google_play_music': /usr/lib/nuvolaplayer3/apprunner -D -a /usr/share/nuvolaplayer3/web_apps/google_play_music 
Runner: [DEBUG      Gtk] Connecting to session manager
Runner: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: gnome shell 3.16.4, CSD 1, appmenu 1, menubar 0
Runner: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
Master: [DEBUG    Nuvola] MasterController.vala:287: Connected to runner server for 'google_play_music'.
Runner: [DEBUG    Nuvola] Connection.vala:132: Network Proxy: system settings
Runner: [DEBUG    Nuvola] WebEngine.vala:65: Nuvola WebKit Extension directory: /usr/lib/nuvolaplayer3
Runner: [WARNING  GLib-GObject] g_object_set_property: construct property "local-storage-directory" for object 'WebKitWebContext' can't be set after construction
Worker: [DEBUG    Nuvola] WebExtension.vala:237: Page 1 created for (null)
Runner: [DEBUG    Nuvola] Notifications.vala:175: Notifications: persistence true, actions true, icons false
Master: [DEBUG    Nuvola] MediaKeys.vala:99: GNOME settings daemon appeared: org.gnome.SettingsDaemon, :1.11
Runner: [DEBUG    Nuvola] MPRISProvider.vala:73: Bus acquired: org.mpris.MediaPlayer2.NuvolaAppGooglePlayMusic, registering objects
Runner: [DEBUG    Nuvola] MPRISProvider.vala:90: Bus name acquired: org.mpris.MediaPlayer2.NuvolaAppGooglePlayMusic
Runner: [DEBUG    Nuvola] WebEngine.vala:462: Navigation, current window: uri = https://play.google.com/music/listen#/albums, result = true, frame = (null), type = WEBKIT_NAVIGATION_TYPE_OTHER
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Trying to play /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from NULL to READY.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from READY to PAUSED.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from PAUSED to PLAYING.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: End of stream for file /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: iTunes Application Detector (enabled, not flash) at /usr/lib/mozilla/plugins/librhythmbox-itms-detection-plugin.so: This plug-in detects the presence of iTunes when opening iTunes Store URLs in a web page with Firefox.
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Gnome Shell Integration (enabled, not flash) at /usr/lib/mozilla/plugins/libgnome-shell-browser-plugin.so: This plugin provides integration with Gnome Shell for live extension enabling and disabling. It can be used only by extensions.gnome.org
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Shockwave Flash (enabled, flash) at /usr/lib/mozilla/plugins/flashplugin-alternative.so: Shockwave Flash 11.2 r202
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Java(TM) Plug-in 10.80.2 (enabled, not flash) at /usr/lib/mozilla/plugins/libjavaplugin.so: Next Generation Java Plug-Runner: in 10.80.2 for Mozilla browsers
Runner: [DEBUG    Nuvola] WebAppWindow.vala:170: Sidebar position: 1409
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [INFO     Nuvola] AppRunnerController.vala:236: Logged in as  ()
```

**Comment by wollac11** [comment]
I also have the same issue (also Ubuntu Gnome 15.10). I feel it happens slightly less frequently since the last update (before it blank was every time on the first launch at least and sometimes also on one or more subsequent launches) but it still happens with the latest version sometimes.

**Comment by bitdivision** [comment]
I too have the same issue.

Arch Linux with i3 window manager.

Nuvola 3.0.0.r15

Debug Output: 

```
Master: [DEBUG      Gtk] Connecting to session manager
Master: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: i3 (null), CSD 0, appmenu 0, menubar 0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.12
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_test 48 not found.
Master: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.12
Master: [INFO     Nuvola] WebAppRegistry.vala:89: Using web app Google Play Music, version 5.12, data dir /usr/share/nuvolaplayer3/web_apps/google_play_music
Master: [DEBUG    Nuvola] MasterController.vala:463: Launch app runner for 'google_play_music': /usr/lib/nuvolaplayer3/apprunner -D -a /usr/share/nuvolaplayer3/web_apps/google_play_music 
Runner: [DEBUG      Gtk] Connecting to session manager
Runner: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: i3 (null), CSD 0, appmenu 0, menubar 0
Runner: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
Master: [DEBUG    Nuvola] MasterController.vala:287: Connected to runner server for 'google_play_music'.
Runner: [DEBUG    Nuvola] Connection.vala:132: Network Proxy: system settings
Runner: [DEBUG    Nuvola] WebEngine.vala:65: Nuvola WebKit Extension directory: /usr/lib/nuvolaplayer3
Runner: [WARNING  GLib-GObject] g_object_set_property: construct property "local-storage-directory" for object 'WebKitWebContext' can't be set after construction
Runner: [DEBUG    Nuvola] Notifications.vala:175: Notifications: persistence false, actions true, icons false
Master: [DEBUG    Nuvola] MediaKeys.vala:130: GNOME settings daemon vanished: org.gnome.SettingsDaemon
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioPause, count 1
Worker: [DEBUG    Nuvola] WebExtension.vala:237: Page 1 created for (null)
Runner: [DEBUG    Nuvola] MPRISProvider.vala:73: Bus acquired: org.mpris.MediaPlayer2.NuvolaAppGooglePlayMusic, registering objects
Runner: [DEBUG    Nuvola] MPRISProvider.vala:90: Bus name acquired: org.mpris.MediaPlayer2.NuvolaAppGooglePlayMusic
Runner: [DEBUG    Nuvola] WebEngine.vala:462: Navigation, current window: uri = https://play.google.com/music/, result = true, frame = (null), type = WEBKIT_NAVIGATION_TYPE_OTHER
Runner: Error loading module '/usr/lib/mozilla/plugins/nppdf.so': /usr/lib/mozilla/plugins/nppdf.so: wrong ELF class: ELFCLASS32
Runner: Error scanning plugin /usr/lib/mozilla/plugins/nppdf.so, /usr/lib/webkit2gtk-4.0/WebKitPluginProcess returned 256 exit status
Runner: [DEBUG    Nuvola] WebAppWindow.vala:170: Sidebar position: 809
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [INFO     Nuvola] AppRunnerController.vala:236: Logged in as  ()
Runner: Error scanning plugin /home/bitdivision/.mozilla/plugins/npwrapper.npica.so, /usr/lib/webkit2gtk-4.0/WebKitPluginProcess returned 256 exit status
Runner: Error loading module '/usr/lib64/mozilla/plugins/nppdf.so': /usr/lib64/mozilla/plugins/nppdf.so: wrong ELF class: ELFCLASS32
Runner: Error scanning plugin /usr/lib64/mozilla/plugins/nppdf.so, /usr/lib/webkit2gtk-4.0/WebKitPluginProcess returned 256 exit status
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Trying to play /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from NULL to READY.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from READY to PAUSED.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from PAUSED to PLAYING.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: End of stream for file /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Shockwave Flash (enabled, flash) at /usr/lib64/mozilla/plugins/libflashplayer.so: Shockwave Flash 11.2 r202
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Shockwave Flash (enabled, flash) at /usr/lib/mozilla/plugins/libflashplayer.so: Shockwave Flash 11.2 r202
Runner: [WARNING  Nuvola] FormatSupportCheck.vala:158: Too many Flash plugins have been found: 2
```

**Comment by fenryxo** [comment]
Hello all. Thanks for your feedback. I'll probably get around to this issue in April as the fundraiser of March has been quite poor and I've already spent most of the funds working on MPRIS rating #204 & ownCloud script #52. Thanks for your understanding.

**Comment by fenryxo** [comment]
Hello everybody. I had little time to look at this issue and I might have fixed it. It's hard to say as the issue has been really rare on my system. Anyway, the tentative fix is currently included in the **Nuvola Player 3.1.x** Rolling Releases and is going to be also included in the **Nuvola Player 3.0.2** Stable Release at the end of April unless there is any regression.

**Comment by bitdivision** [comment]
@fenryxo Looks like this is fixed for me!

Thanks very much.

**Comment by scotte** [comment]
I've been plagued by this issue as well. I'll hand apply the patch and see if it solves it for me as well. Thanks!

**Comment by fenryxo** [comment]
Fixed in Nuvola Player 3.0.2 Stable Release and in Nuvola Player 3.1 Rolling Releases.
